build/NuGetAdditionalFiles/Microsoft.NETCore.Compilers.props

```diff
@@ -9,15 +9,17 @@
              AssemblyFile="$(MSBuildThisFileDirectory)..\tools\Microsoft.Build.Tasks.CodeAnalysis.dll" />
   <UsingTask TaskName="Microsoft.CodeAnalysis.BuildTasks.CopyRefAssembly"
              AssemblyFile="$(MSBuildThisFileDirectory)..\tools\Microsoft.Build.Tasks.CodeAnalysis.dll" />
+
   <PropertyGroup>
+    <!-- Don't use the compiler server by default in the NuGet package. -->
+    <UseSharedCompilation Condition="'$(UseSharedCompilation)' == ''">false</UseSharedCompilation>
```

Comment:
Why are we making this change? We made this "true" by default in the NuGet package in response to customer feedback here around performance.

Reply:
I don't think that's true -- this matches the desktop behavior as far as I can see.

Reply:
Any projects using the "new" project format will have `UseSharedCompilation` set to `true` by default: https://github.com/dotnet/sdk/blob/8a24663345cc5952062cb898431dbd8800b4da85/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.targets#L428

Reply:
https://github.com/dotnet/roslyn/blob/6db04109879e42fbe99a3c1e8f1d44175c041e73/build/NuGetAdditionalFiles/Microsoft.Net.Compilers.props#L31

I'm not sure why the line above says "in Visual Studio" -- this applies to the NuGet package.

Reply:
@tannergooding That line won't be evaluated until after we evaluate this property if the package is installed via NuGet.

Reply:
Yes, but this line will prevent the compiler server from being used on new SDK which has a different default from Desktop.

Reply:
@tannergooding The same property is set on Desktop: https://github.com/dotnet/roslyn/blob/6db04109879e42fbe99a3c1e8f1d44175c041e73/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L79